project-base/storefront/components/Pages/Order/ContactInformation/FormBlocks/ContactInformationDeliveryAddress.tsx

```diff
@@ -277,8 +277,8 @@ export const ContactInformationDeliveryAddress: FC = () => {
 
                 {!!pickupPlace && (
                     <div>
```

Comment:
I would add some margin above the text for better visibility when the checkbox is not checked.

Reply:
Sure thing, added